dcc-portal-api/src/main/java/org/icgc/dcc/portal/util/Filters.java

```diff
@@ -106,14 +106,18 @@ public static ObjectNode pathwayFilter() {
 
   public static ObjectNode inputGeneSetIdFilter(@NonNull String inputGeneSetId) {
     val geneFilter = geneFilter();
-    geneFilter.with("gene").set(API_ENTITY_SET_ID_FIELD_NAME, is(inputGeneSetId));
+    geneFilter.with("gene").set("id", is("ES:" + inputGeneSetId));
```

Comment:
Same point here about the constant. Also, should probably use a constant for `"id"`